templates/code.template

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+{{generatedString}}
+'''
+{{api['title']}}
+'''
+from __future__ import absolute_import, division, print_function
+
+import logging
+import taskcluster.baseclient as baseclient
+
+log = logging.getLogger(__name__)
+
+
+class {{serviceName}}(baseclient.BaseClient):
+    '''
+    {{api['title']}}
+    {{api['description'] | docstring}}
+    '''
+    version = {{api['version']}}
+    referenceUrl = '{{referenceUrl}}'
+{%- if createRoutes(api) %}
+    routes = {
```

Comment:
Curious why these are class variables instead of being baked into the stub methods as literals.